### ISSUE TITLE
(Suggestion) Allow swiping between dates in ListTimetable

### ISSUE THREAD

**Issue by kosenda** [DroidKaigi/conference-app-2024#606]
## Overview
On devices with large screens, I thought it would be a bit of a hassle to switch the date tabs because they are located at the very top of the screen, so I thought it would be nice if they could be switched by swiping.

## Sample
If we use `draggable`, an extension function of Modifier

https://github.com/user-attachments/assets/49451168-67c0-4e34-9385-4ef09b508372

**Comment by takahirom** [comment]
@nobonobopurin Do you have any thoughts on this? Instead of this, we could use a view pager or something similar.

**Comment by nobonobopurin** [comment]
@takahirom @kosenda Yes, please! I’d love to see swiping between dates in ListTimetable.

**Comment by nobonobopurin** [comment]
@kosenda cc @takahirom 
To clarify, instead of switching to the right tab with a gesture, imagine the tab switching following the finger as it slides horizontally. 

Reference URL：
[](https://m3.material.io/styles/motion/transitions/transition-patterns#62c97bc6-e7e8-4b86-88f3-9c1b831a7c9c)

**Comment by takahirom** [comment]
Thanks! Let's unblock this issue!

**Comment by gargVader** [comment]
👋 Can I take this up?

**Comment by takahirom** [comment]
Thanks! Assgined👍
